lib/index.js

```diff
@@ -80,28 +86,29 @@ exports.balm = (opts = {}) => {
     console.log('Balm watching...');
 
     _socket = new WebSocket.Server({ port });
-    addHmrToBundles(port, blocksDir);
-
-    const watcher = 'chokidar';
-    const chokidar = require(watcher);
-    const distWatchOptions = {
-      awaitWriteFinish: {
-        stabilityThreshold: 100,
-        pollInterval: 25
-      }
-    };
-    const bundles = chokidar.watch(['./public/dist/*.css', './public/dist/*.js'], distWatchOptions);
-    const balmFiles = chokidar.watch(`${root}/**/*.balm`);
 
-    bundles.on('change', (path) => {
-      if (hasBundled[path]) {
+    fs.watch('./public/dist', (_, path) => {
+      if (!hasBundled[path]) {
+        console.log('File changed for HMR', path);
         notifyClient(path);
-      } else {
         hasBundled[path] = true;
       }
     });
 
-    balmFiles.on('change', bundleFile);
+    if (!fs.existsSync('assets-map.json')) {
+      console.log('Creating assets-map.json');
+      fs.writeFileSync('assets-map.json', '', 'utf8');
+    }
+
```

Comment:
Why not just do:

```
fs.writeFileSync('assets-map.json', '', 'utf8');
```

Every time?

Reply:
Yeah, you're right. With the "building in app, in even in production" approach, we needed the check. I forgot to take it out once we changed that.